### PR TITLE
Migrate chip

### PR DESCRIPTION
**Pull request by ghsteff**
https://github.com/reaviz/reablocks/assets/40581813/2709ce8f-38af-4c6a-908d-0da25840cdb8

### CODE REVIEW

File: src/elements/Chip/DeletableChip.tsx [hunk 1]
```diff
@@ -18,17 +18,23 @@ export interface DeletableChipProps extends Omit<ChipProps, 'end'> {
...
           tabIndex={0}
           variant="text"
           size={size}
-          className={classNames(css.deleteButton, { [css[size]]: true })}
+          className={classNames(
+            css.deleteButton,
+            { [css[size]]: true },
+            { [`text-${color}-500`]: rest?.variant === 'outline' },
+            { ['text-white']: rest?.variant === 'filled' }
+          )}
```
Comment (ghsteff) [inline src/elements/Chip/DeletableChip.tsx:37]: this is maybe a little hacky, but it ensures the close button matches the color when using the outline variant
image

Comment (ghsteff) [reply src/elements/Chip/DeletableChip.tsx]: nvm replaced this with `text-inherit`

**Review by SerhiiTsybulskyi — APPROVED** [review]
few minor things

**Comment by amcdnl** [comment]
@SerhiiTsybulskyi - Can you move this code into your PR and close this one?

File: src/elements/Chip/Chip.tsx [hunk 2]
```diff
@@ -72,31 +73,55 @@ export const Chip: FC<ChipProps & ChipRef> = forwardRef(
...
+          theme.base,
+          theme.colors[color]?.base,
+          theme.sizes[size],
+          theme.focus,
+          !!onClick && !disabled && theme.colors[color]?.selectable,
+          selected && theme.colors[color]?.selected,
+          theme.variants[variant],
+          disableMargins && 'm-0',
+          css.chip,
+          'transition-colors duration-300 ease',
```
[the remark this answers is not in the document]
Comment (ghsteff) [reply src/elements/Chip/Chip.tsx:95]: oh yea true thanks

**Pull request merged**